tests/Unit/Evolution/Systems/NewtonianEuler/BoundaryCorrections/Test_Hll.cpp

```diff
@@ -0,0 +1,174 @@
+  using packed_type = bool;
+
+  static inline unpacked_container unpack(
+      const packed_container& /*packed*/,
+      const size_t /*grid_point_index*/) noexcept {
+    return true;
+  }
+
+  [[noreturn]] static inline void pack(
+      const gsl::not_null<packed_container*> /*packed*/,
+      const unpacked_container& /*unpacked*/,
+      const size_t /*grid_point_index*/) {
+    ERROR("Should not be converting an EOS from an unpacked to a packed type");
+  }
+
+  static inline size_t get_size(const packed_container& /*packed*/) noexcept {
+    return 1;
+  }
+};
+
+struct ConvertIdeal {
+  using unpacked_container = bool;
+  using packed_container = EquationsOfState::IdealFluid<false>;
+  using packed_type = bool;
+
+  static inline unpacked_container unpack(
+      const packed_container& /*packed*/,
+      const size_t /*grid_point_index*/) noexcept {
+    return false;
+  }
+
+  [[noreturn]] static inline void pack(
+      const gsl::not_null<packed_container*> /*packed*/,
+      const unpacked_container& /*unpacked*/,
+      const size_t /*grid_point_index*/) {
+    ERROR("Should not be converting an EOS from an unpacked to a packed type");
+  }
+
+  static inline size_t get_size(const packed_container& /*packed*/) noexcept {
+    return 1;
+  }
+};
+
+struct DummyInitialData {
+  using argument_tags = tmpl::list<>;
+  struct source_term_type {
+    using sourced_variables = tmpl::list<>;
+    using argument_tags = tmpl::list<>;
+  };
+};
+
+namespace helpers = TestHelpers::evolution::dg;
+
+template <size_t Dim, typename EosType>
+void test(const gsl::not_null<std::mt19937*> gen, const size_t num_pts,
+          const EosType& equation_of_state) {
+  tuples::TaggedTuple<hydro::Tags::EquationOfState<EosType>> volume_data{
+      equation_of_state};
+  tuples::TaggedTuple<
+      helpers::Tags::Range<NewtonianEuler::Tags::MassDensityCons>,
+      helpers::Tags::Range<
+          NewtonianEuler::Tags::SpecificInternalEnergy<DataVector>>>
+      ranges_for_conservation{std::array<double, 2>{{1.0e-30, 1.0}},
+                              std::array<double, 2>{{1.0e-30, 1.0}}};
+
+  helpers::test_boundary_correction_conservation<
+      NewtonianEuler::System<Dim, EosType, DummyInitialData>>(
+      gen, NewtonianEuler::BoundaryCorrections::Hll<Dim>{},
+      Mesh<Dim - 1>{num_pts, Spectral::Basis::Legendre,
+                    Spectral::Quadrature::Gauss},
+      volume_data, ranges_for_conservation);
+
+  tuples::TaggedTuple<
+      helpers::Tags::Range<NewtonianEuler::Tags::MassDensityCons>,
+      helpers::Tags::Range<
+          NewtonianEuler::Tags::SpecificInternalEnergy<DataVector>>,
+      helpers::Tags::Range<typename NewtonianEuler::BoundaryCorrections::Hll<
+          Dim>::LargestOutgoingCharSpeed>,
+      helpers::Tags::Range<typename NewtonianEuler::BoundaryCorrections::Hll<
+          Dim>::LargestIngoingCharSpeed>>
+      ranges_for_python{std::array<double, 2>{{1.0e-30, 1.0}},
```

Comment:
if you decide you like the shorter notation, please use it here too so that the file is consistent

Reply:
Okay! I I find your suggestion great since that comes with less number of curly braces :)